src/v/redpanda/admin/api-doc/cluster.json

```diff
@@ -108,6 +108,10 @@
                         "type": "string"
                     },
                     "description": "list of partitions where one or more replicas has not replicated all data"
+                },
+                "bytes_in_cloud_storage": {
```

Comment:
so this is just from last window? where do we store the 24 buckets history?

Reply:
no this type is the type of a bucket, there are N buckets stored historically